.github/workflows/release.yml

```diff
@@ -281,11 +291,23 @@ jobs:
           echo "---" >> klt-cert-manager/config/rendered/release.yaml
           cat klt-cert-manager/config/rendered/release.yaml > cert-manager-manifest.yaml
 
-      - name: Attach KLT release assets
-        if: needs.release-please.outputs.klt-release-created == 'true'
+      - name: Create Metrics Operator manifest
+        if: needs.release-please.outputs.metrics-operator-release-created == 'true'
+        env:
+          RELEASE_REGISTRY: ghcr.io/keptn
+          CHART_APPVERSION: ${{ needs.release-please.outputs.metrics-operator-tag-name }}
+        run: |
+          cd metrics-operator
+          make controller-gen release-manifests
+          cd ..
+          echo "---" >> metrics-operator/config/rendered/release.yaml
```

Comment:
is the namespace added in the makefile?

Reply:
probably not, I will check. I think we need to add the namespace here and also the the cert-manager manifest. It's hard to test these things...Good catch!

Reply:
fixed